### PR TITLE
Fix Calendar/DateRangePicker overflow in docs/stories

### PR DESCRIPTION
**Pull request by LFDanLu**
Closes 

## ✅ Pull Request Checklist:

- [ ] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [ ] Updated documentation (if it already exists for this component).
- [ ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)

## 🧢 Your Project:

RSP

### CODE REVIEW

**Review by snowystinger — COMMENTED** [review]
I'm not sure this the way we should solve the zoom issue
Probably better to apply 100% (vw may cause scroll bars to appear) width on things

Can we apply anything to the entire docs pages? instead of directly on the date range pickers
Like you do for the stories

**Review by ktabors — COMMENTED** [review]
I thought we were applying the change to the width of the DateRangePicker input field. Something like changing the `width: auto` at https://github.com/adobe/react-spectrum/blob/556773fefe1760473111e94ae551119b27d538cb/packages/%40react-spectrum/datepicker/src/index.css#L36 to `max-width: 100%`.

**Comment by LFDanLu** [comment]
@snowystinger I guess I could wrap each problematic DateRangePicker in a View that constraints the width, but just wanna clarify you point about `vw`: is the concern here that at some screen sizes the `vw` may overflow still? 

@ktabors I think `width: auto` is correct as default here since it will take into account borders and stuff when growing to match its parent container, but not 100% sure

**Comment by snowystinger** [comment]
my concern with vw is that doesn't account for the width of a scrollbar, so it can cause scrollbars that you don't intend.

**Review by ktabors — COMMENTED** [review]
LGTM

**Review by ktabors — APPROVED** [review]

File: packages/@react-spectrum/calendar/stories/Calendar.stories.tsx [hunk 1]
```diff
@@ -148,7 +149,9 @@ function Example(props) {
         </Picker>
       </Flex>
       <Provider locale={(locale || defaultLocale) + (calendar && calendar !== preferredCalendars[0].key ? '-u-ca-' + calendar : '')}>
-        <Calendar {...props} />
+        <View maxWidth="100vw" overflow="auto">
```
[the remark this answers is not in the document]
Comment (LFDanLu) [reply packages/@react-spectrum/calendar/stories/Calendar.stories.tsx:152]: What do you see in the visibleMonths story? If you can now scroll the calendars then the above change is working, previously the calendar overflowed the story container

**Review by reidbarber — APPROVED** [review]

**Review by devongovett — APPROVED** [review]

**Pull request merged**